src/mqt/bench/evaluation/evaluation.py

```diff
@@ -57,3 +56,12 @@ def evaluate_qasm_file(filename: str) -> EvaluationResult:
         "parallelism": parallelism,
         "liveness": liveness,
     }
+    return cast(EvaluationResult, res_dict)
+
+
+def count_occurrences(filenames: list[str], search_str: str) -> int:
+    return sum([search_str in filename for filename in filenames])
+
+
+def count_qubit_numbers_per_compiler(filenames: list[str], compiler: str) -> list[int]:
+    return [int(str(filename).split("_")[-1].split(".")[0]) for filename in filenames if compiler in filename]
```

Comment:
These functions aren't covered by tests. That should be fairly easy to change.

Reply:
Added respective tests.